player/loadfile.c

```diff
@@ -300,19 +297,45 @@ static void print_stream(struct MPContext *mpctx, struct track *t)
         }
     } else if (t->type == STREAM_AUDIO) {
         if (s && s->codec->channels.num)
-            APPEND(b, " %d ch", s->codec->channels.num);
-        if (s && s->codec->samplerate) {
-            char *samplerate = mp_format_double(NULL, s->codec->samplerate / 1000.0,
-                                                4, false, false, true);
-            APPEND(b, " %s kHz", samplerate);
-            talloc_free(samplerate);
-        }
+            APPEND(b, " %dch", s->codec->channels.num);
+        if (s && s->codec->samplerate)
+            APPEND(b, " %d Hz", s->codec->samplerate);
     }
-    APPEND(b, ")");
     if (s && s->hls_bitrate > 0)
-        APPEND(b, " (%d kbps)", (s->hls_bitrate + 500) / 1000);
-    if (t->is_external)
-        APPEND(b, " (external)");
+        APPEND(b, "  %d kbps", (s->hls_bitrate + 500) / 1000);
+    APPEND(b, ")");
+
+    bool first = true;
+    if (t->default_track || t->forced_track || t->attached_picture ||
+        t->visual_impaired_track || t->hearing_impaired_track || t->is_external)
+        APPEND(b, " [");
```

Comment:
```suggestion
    if (t->default_track || t->forced_track || t->attached_picture ||
        t->visual_impaired_track || t->hearing_impaired_track || t->is_external)
    {
        APPEND(b, " [");
    }
```

Reply:
or even move it to `first` check 

`APPEND(b, first ? " [" : ""); first = false;`